assets/styles/style.scss

```diff
@@ -615,12 +615,16 @@ body.dark-theme {
 
 
 .email-container {
+    height: 100vh;
```

Comment:
This should better be `height: 100%` because we really want this element to be 100% of the height of its parent. That would account for possible future changes in global layout.